config/settings.py

```diff
@@ -281,3 +281,8 @@
 # ------------------------------------------------------------------------------
 
 GOOGLE_CLOUD_VISION_API_KEY = os.getenv("GOOGLE_CLOUD_VISION_API_KEY")
+
+# Google Gemini API
+# ------------------------------------------------------------------------------
+
+GOOGLE_GEMINI_API_KEY = os.getenv("GOOGLE_GEMINI_API_KEY")
```

Comment:
@teolemon how do GOOGLE API works, could we re-use the GOOGLE_CLOUD_VISION_API_KEY ?
If not, could you add OFF's GOOGLE_GEMINI_API_KEY somewhere so that when we deploy in staging/prod the endpoint works ? 
Thanks !!

Reply:
Those are scoped, if we're using a specific key for Open Prices, we can extend the scope. Otherwise, @raphael0202 may have a key dedicated to Gen AI.